src/main/java/core/basesyntax/figuresuppliers/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.figuresuppliers;
+
+import core.basesyntax.color.Color;
+import core.basesyntax.color.ColorSupplier;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    static final Color DEFAULT_COLOR = Color.WHITE;
+    static final int MAX_EDGE = 10;
+    static final int ANNEX = 1;
+    static final int DEFAULT_RADIUS = 10;

```

Comment:
Please pay attention to the access modifiers. By setting no modifiers you implicitly set package-private modifier, do we really need to open these variables for other classes in this package?